packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -249,7 +253,8 @@ class Auth {
             refresh_token: this.get('refresh_token_registered') || this.get('refresh_token_guest'),
             token_type: this.get('token_type'),
             usid: this.get('usid'),
-            customer_type: this.get('customer_type') as CustomerType
+            customer_type: this.get('customer_type') as CustomerType,
+            refresh_token_expires_in: this.get('refresh_token_expires_in')
```

Comment:
@shethj can you point me to where in the implementation this value gets set by the API? We're expecting the API to set this rather than dynamically setting it right?

Reply:
The value is returned from the getAccessToken API in the response body. We read that value and set it to storage here: 
https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1459/files#diff-4f3bb7407698d0a81a28ff00462f1589482fa8e984b6b19622286c5f6db36bb1R122